src/fabric/SvgViewNativeComponent.ts

```diff
@@ -0,0 +1,25 @@
+// @ts-ignore TS being TS
+import codegenNativeComponentUntyped from 'react-native/Libraries/Utilities/codegenNativeComponent';
+import type { ViewProps, HostComponent, ColorValue } from 'react-native';
+// @ts-ignore TS being TS
+import type { Float, Int32 } from 'react-native/Libraries/Types/CodegenTypes';
+
+interface NativeProps extends ViewProps {
+  bbWidth: string,
+  bbHeight: string,
+  minX: Float,
+  minY: Float,
+  vbWidth: Float,
+  vbHeight: Float,
+  align: string,
+  meetOrSlice: Int32,
+  tintColor: ColorValue,
+  color: ColorValue,
+}
+
+// eslint-disable-next-line @typescript-eslint/ban-types
+const codegenNativeComponent = codegenNativeComponentUntyped as <T extends {}>(
+  name: string
+) => HostComponent<T>;
+
+export default codegenNativeComponent<NativeProps>('RNSVGSvgView');
```

Comment:
It would be nice thing to have all NativeComponent interfaces declared in the same language (all in Flow / TS). This one stands out.